packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -205,25 +213,40 @@ export class IndexedDbPersistence implements Persistence {
 
   private queryCache: IndexedDbQueryCache;
   private remoteDocumentCache: IndexedDbRemoteDocumentCache;
+  private webStorage?: Storage;
+  private listenSequence: ListenSequence;
 
   constructor(
     private readonly persistenceKey: string,
     private readonly clientId: ClientId,
     platform: Platform,
     private readonly queue: AsyncQueue,
     serializer: JsonProtoSerializer,
-    synchronizeTabs: boolean
+    private readonly multiClientParams?: {
+      sequenceNumberSyncer: SequenceNumberSyncer;
+    }
   ) {
     this.dbName = persistenceKey + IndexedDbPersistence.MAIN_DATABASE;
     this.serializer = new LocalSerializer(serializer);
     this.document = platform.document;
     this.window = platform.window!;
-    this.allowTabSynchronization = synchronizeTabs;
+    if (multiClientParams !== undefined) {
+      this.allowTabSynchronization = true;
+    } else {
+      this.allowTabSynchronization = false;
+    }
     this.queryCache = new IndexedDbQueryCache(this.serializer);
     this.remoteDocumentCache = new IndexedDbRemoteDocumentCache(
       this.serializer,
       /*keepDocumentChangeLog=*/ this.allowTabSynchronization
     );
+    this.webStorage = this.window.localStorage;
+    if (!this.webStorage) {
```

Comment:
Since we mock WebStorage now as well, can you verify if we still need this?

Reply:
Verified that the mock persistence works. I can't test browsers locally anymore, so i'm delegating that to travis